### PR TITLE
perf(ui): Move organization level stream to main bundle

### PR DESCRIPTION
**Pull request by lynnagara**
Also moves the old project based stream out of the main bundle. Since
sentry 10 is default for everyone, very few users will load the old stream
anymore.

### CODE REVIEW

File: src/sentry/static/sentry/app/routes.jsx [hunk 1]
```diff
@@ -1056,10 +1049,24 @@ function routes() {
         </Route>
 
         <Route path=":projectId/" component={errorHandler(ProjectDetails)}>
-          <IndexRoute component={errorHandler(Stream)} />
-          <Route path="issues/" component={errorHandler(Stream)} />
+          <IndexRoute
+            componentPromise={() =>
+              import(/*webpackChunkName: "Stream"*/ './views/stream')}
```
Comment (markstory) [inline src/sentry/static/sentry/app/routes.jsx]:
```suggestion
              import(/* webpackChunkName: "Stream" */ './views/stream')}
```

Are the spaces important in these comments?

File: src/sentry/static/sentry/app/routes.jsx [hunk 2]
```diff
@@ -1056,10 +1049,24 @@ function routes() {
...
+            component={errorHandler(LazyLoad)}
+          />
+          <Route
+            path="issues/"
+            componentPromise={() =>
+              import(/*webpackChunkName: "Stream"*/ './views/stream')}
```
Comment (markstory) [inline src/sentry/static/sentry/app/routes.jsx]:
```suggestion
              import(/* webpackChunkName: "Stream" */ './views/stream')}
```

File: src/sentry/static/sentry/app/routes.jsx [hunk 3]
```diff
@@ -1056,10 +1049,24 @@ function routes() {
...
+            component={errorHandler(LazyLoad)}
+          />
 
-          <Route path="searches/:searchId/" component={errorHandler(Stream)} />
+          <Route
+            path="searches/:searchId/"
+            componentPromise={() =>
+              import(/*webpackChunkName: "Stream"*/ './views/stream')}
```
Comment (markstory) [inline src/sentry/static/sentry/app/routes.jsx]:
```suggestion
              import(/* webpackChunkName: "Stream" */ './views/stream')}
```

File: src/sentry/static/sentry/app/routes.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/routes.jsx]: I'm actually not 100% sure, adding anyway though

**Review by markstory — APPROVED** [review]

**Pull request merged**